src/datetime/timeAgo.test.ts

```diff
@@ -72,7 +72,7 @@ describe('timeAgo', () => {
 		const yesterday = new Date(Date.UTC(2019, 10, 16, 3, 0, 0)).getTime();
 
 		expect(timeAgo(yesterday)).toBe('1d ago');
-		expect(timeAgo(yesterday, { verbose: true })).toBe('Yesterday 3:00');
+		expect(timeAgo(yesterday, { verbose: true })).toBe('Yesterday 3.00');
```

Comment:
The style guide seems to suggest this should omit the minutes and opt for `Yesterday 3`

Reply:
i'd be interested to know what UX think. intuitively it feels like we have multiple use cases here. a journalist writing "yesterday at 1am" seems very different to a status of "15.00". the `.` though def seems to fit the style guide.